control/ipv6.yaml

```diff
@@ -0,0 +1,113 @@
+            ping:
+              x-field-uid: 1
+          x-field-uid: 1
+        ping:
+          $ref: "#/components/schemas/Action.Protocol.Ipv6.Ping"
+          x-field-uid: 2
+    Action.Response.Protocol.Ipv6:
+      description: >-
+        Response for actions associated with IPv6 on configured resources.
+      type: object
+      required:
+        - choice
+      properties:
+        choice:
+          type: string
+          x-enum:
+            ping:
+              x-field-uid: 1
+          x-field-uid: 1
+        ping:
+          $ref: "#/components/schemas/Action.Response.Protocol.Ipv6.Ping"
+          x-field-uid: 2
+    Action.Protocol.Ipv6.Ping:
+      description: Request for initiating ping between multiple source and destination pairs.
+      type: object
+      properties:
+        requests:
+          description: >-
+            List of IPv6 ping requests.
+          type: array
+          items:
+            $ref: "#/components/schemas/Action.Protocol.Ipv6.PingRequest"
+          x-field-uid: 1
+    Action.Protocol.Ipv6.PingRequest:
+      description: >-
+        Request for initiating ping between a single source and destination pair.
+
+        For ping request, 1 IPv6 ICMP Echo Request shall be sent and wait for ping response to
+        either succeed or time out. The API wait timeout for each request shall be 300ms.
+      type: object
+      properties:
+        src_name:
+          description: >-
+            Name of source IPv6 interface to be used.
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv6/properties/name'
+          x-field-uid: 1
+        dst_ip:
+          description: >-
+            Destination IPv6 address to ping.
+          type: string
+          format: ipv6
+          x-field-uid: 2
+    Action.Response.Protocol.Ipv6.Ping:
+      description: >-
+        Response for ping initiated between multiple source and destination pairs.
+      type: object
+      properties:
+        responses:
+          description: >-
+            List of responses for IPv6 ping responses.
+          type: array
+          items:
+            $ref: "#/components/schemas/Action.Response.Protocol.Ipv6.PingResponse"
+          x-field-uid: 1
+    Action.Response.Protocol.Ipv6.PingResponse:
+      description: >-
+        Response for ping initiated between a single source and destination pair.
+      type: object
+      properties:
+        src_name:
+          description: >-
+            Name of source IPv6 interface used for ping.
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv6/properties/name'
+          x-field-uid: 1
+        dst_ip:
+          description: >-
+            Destination IPv6 address used for ping.
+          type: string
+          format: ipv6
+          x-field-uid: 2
+        result:
+          description: >-
+            Result of the ping request.
+          type: string
+          default: failed
```

Comment:
Same as earlier.  Why `failed` as the default?

Reply:
thanks for pointing out; removed defaults and marked all three parameters as `required`.